dash/development/_r_components_generation.py

```diff
@@ -331,62 +355,49 @@ def write_help_file(name, props, description, prefix):
     default_argtext += ", ".join("{}=NULL".format(p) for p in prop_keys)
 
     item_text += "\n\n".join(
-        "\\item{{{}}}{{{}{}}}".format(p,
-                                      print_r_type(
-                                          props[p]["type"]
-                                      ),
-                                      props[p]["description"])
+        "\\item{{{}}}{{{}{}}}".format(
+            p,
+            print_r_type(props[p]["type"]),
+            props[p]["description"]
+        )
         for p in prop_keys
     )
 
-    if has_wildcards:
-        item_text += '\n\n\\item{...}{wildcards: `data-*` or `aria-*`}'
+    if any(key.endswith("-*") for key in prop_keys):
         default_argtext += ', ...'
+        item_text += wildcard_help_template.format(get_wildcards_r(prop_keys))
```

Comment:
🐄  minor: always try to use list for str operation `+=` might be expensive if it's in a big loop